docs/content/dependencies-file.md

```diff
@@ -142,6 +142,21 @@ nuget Example >= 2.0
 If you change the target framework of the projects then you need to run
 [`paket install`](paket-install.html) again.
 
+### Disable packages folder
+
+You can opt-out of generating the `packages` folder by using the `storage` option:
+
+
+```paket
+// Do not extract into the "packages" folder but use a globally shared directory
+storage: none
```

Comment:
we should discuss how this should look like. `storage` might be a bit too generic.

Technically we could allow the user so specify a custom path now. We basically have everything ready but I don't allow it as part of this PR, we first need to decide how that should look (considering x-plat for example).